liota/core/package_manager.py

```diff
@@ -641,9 +659,13 @@ def _package_load(self, file_name, checksum=None, autoload_flag=False,
                         break
                 if (verify_flag == False):
                     log.error("Package %s integrity verification failed" % path_file_ext)
+                    with open(self._rsp_pipe_file, "w+") as fp:
+                        fp.write('Failure\n')
                     return None
             else:
                 sha1 = sha1sum(path_file_ext);
+                with open(self._rsp_pipe_file, "w+") as fp:
+                    fp.write('Success\n')
```

Comment:
667/668: Same comment as above. Even more so since we're using the Success string again.

Reply:
done the same as above